.docker/docker-compose.yml

```diff
@@ -6,3 +6,12 @@ services:
       context: ../
     ports:
       - '8080:8080'
+  console:
+    image: tpluscode/hypermedia-app:alpha
+    environment:
+      - 'API_ENTRYPOINTS={ "http://localhost:8080/": "Article Store" }'
+      - 'BASE_URL=http://localhost:8080/'
```

Comment:
Not working due to https://github.com/hypermedia-app/generic.hypermedia.app/issues/44.